#! /bin/sh /usr/share/dpatch/dpatch-run
## 38_mips64_build.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch from Martin Michlmayr for mips64 builds. bz#330626

@DPATCH@

diff --git a/mozilla/security/coreconf/Linux.mk b/mozilla/security/coreconf/Linux.mk
index 3f10f99..d610752 100644
--- a/mozilla/security/coreconf/Linux.mk
+++ b/mozilla/security/coreconf/Linux.mk
@@ -87,6 +87,9 @@ else
 ifeq (,$(filter-out parisc%,$(OS_TEST)))
 	CPU_ARCH        = hppa
 else
+ifeq ($(OS_TEST),mips64)
+	CPU_ARCH        = mips
+else
 ifeq (,$(filter-out i%86,$(OS_TEST)))
 	OS_REL_CFLAGS	+= -Di386
 	CPU_ARCH	= x86
@@ -101,6 +104,7 @@ endif
 endif
 endif
 endif
+endif
 
 
 LIBC_TAG		= _glibc
